patches/tModLoader/Terraria/ModLoader/FrameworkVersion.cs

```diff
@@ -5,39 +5,18 @@
 
 namespace Terraria.ModLoader
```

Comment:
Even with ditching mono, lets keep XNA and net framework for just a little longer. You've done half the work with all the conditionals.

Reply:
Agreed. Not ditching Framework with this change, though; the Framework enum is simply replaced by preprocessor directives.